docs/Tutorials/Send-private-transaction.md

```diff
@@ -0,0 +1,111 @@
+---
+description: Sending private transactions
+---
+
+# Send private transactions
+
+## Prerequisites
+
+* [Privacy-enabled network as configured in tutorial](Create-Privacy-enabled-network.md).
```

Comment:
nodes and tessera configures AND running are required here

Reply:
Fixed